design-guidelines/interactive-element-states.md

```diff
@@ -0,0 +1,67 @@
+# Interactive Element States
+
+We can make our apps more intuitive by communicating the status of an interactive element, and providing feedback.
+
+## Common interaction subjects
+
+- Actions
+  - For example: buttons
+- Selectable items
+  - For example: items in a list of selectable options
+- Data inputs
+  - For example: a text input
+
+## States
+
+### Disabled
+
+Use a disabled state when an element cannot be selected or interacted with, but there is something that can be done to enable the element.
+
+It should be obvious that something is disabled before a user tries to interact with it.
+
+Disabled elements should not provide any feedback when they're clicked.
+
+Disabled elements should not be keyboard-navigable unless a tooltip is displayed when the disabled element is focus.
+
+#### When to disable an element
+
+An interactive element should be disabled when:
+
```

Comment:
- When presenting 2 or more primary options and important to show impacts of not selecting the other options. The unselected option(s) would have their child fields disabled but still visible so the impacts are clear. This is more typical when primary options are radio buttons with child fields.

Reply:
I just realized we already address this case in the first point:

> there is something a user can do in their current session to enable the component.

I think what you're describing is just an example of that behavior. Please explain if there's something I'm missing.